mesheryctl/internal/cli/root/system/restart.go

```diff
@@ -87,6 +85,24 @@ func restart() error {
 			return ErrRestartMeshery(err)
 		}
 	} else {
+		if currPlatform == "kubernetes" {
+			userResponse := false
+			if utils.SilentFlag {
+				userResponse = true
+			} else {
+				// ask user for confirmation
+				userResponse = utils.AskForConfirmation("Meshery deployments will be deleted from your cluster. Are you sure you want to continue")
+			}
+			if !userResponse {
+				log.Info("Restart aborted.")
+				return nil
+			}
+			// skips asking for confirmation in the stop() function
+			utils.SilentFlag = true
```

Comment:
I feel, we should let start command ask the confirmation related to endpoint again, we should not assume the response related to k8s cluster is same as response related to endpoint.